HARK/ConsumptionSaving/ConsIndShockModelFast.py

```diff
@@ -104,14 +103,14 @@ class PerfForesightSolution(MetricObject):
     distance_criteria = ["cNrm", "mNrm"]
 
     def __init__(
-        self,
-        mNrm=np.array([0.0, 1.0]),
-        cNrm=np.array([0.0, 1.0]),
-        vFuncNvrsSlope=0.0,
-        mNrmMin=0.0,
-        hNrm=0.0,
-        MPCmin=1.0,
-        MPCmax=1.0,
+            self,
+            mNrm=np.array([0.0, 1.0]),
+            cNrm=np.array([0.0, 1.0]),
+            vFuncNvrsSlope=0.0,
+            mNrmMin=0.0,
+            hNrm=0.0,
+            MPCmin=1.0,
+            MPCmax=1.0,
```

Comment:
I'm not sure what these extra indents throughout are for. Is this an editor configuration issue?